packages/codegen-ui/lib/types/form/form-cta.ts

```diff
@@ -17,7 +17,7 @@ import { StudioFieldPosition } from './position';
 
 export type StudioFormCTAButton =
   | {
-      excluded: true;
+      excluded: boolean;
```

Comment:
can we keep this so that it's either excluded is `true` or excluded is not there at all? That's how it works for field configs in the `Common` package too. 

So in `map-cta.ts`, it would be 
`element === 'cancel' && !buttons?.cancel`
Meaning if it's `{}`, it does have an override and it should be rendered. That's how it works for field configs too. 
That way, in the UI, customers can include the cancel button without necessarily modifying the label, etc.

For your `shouldIncludeCancel` logic in codegen-ui-react, the same would apply.
Or maybe you can grab the info from the `buttonMatrix` instead if it's available

Reply:
synced offline - updated pr with new changes